examples/rpc-kitchen-sink/index.mjs

```diff
@@ -0,0 +1,53 @@
+import { mkRPC } from '@reach-sh/rpc-client';
+
+(async () => {
+  const { rpc, rpcCallbacks } = await mkRPC();
+
+  const assertEq = async (l, e, a) => {
+    const f = x => JSON.stringify(x).replace(/\\u0000/g, '');
+    const [ ej, aj ] = [ f(e), f(a) ];
+    if (ej === aj) {
+      console.log(`${l} ${ej} === ${aj}`);
+    } else {
+      console.log(`${l} *** Mismatch! ***`);
+      console.log(`  Expected: ${ej}`);
+      console.log(`    Actual: ${aj}`);
+    }
+    await rpc('/stdlib/assert', ej === aj);
+  };
+
+  const accAlice = await rpc('/stdlib/newTestAccount', await rpc('/stdlib/parseCurrency', 100));
+  const ctcAlice = await rpc('/acc/contract', accAlice);
+  const meta     = "This is a test string";
+
+  /* Views */
+  const checkViewFun = async (l, x, y, e) => {
+    await assertEq(`ctc.v     > ${l}:`, e, await rpc(`/ctc/v/${l}`,     ctcAlice, x, y));
+    await assertEq(`ctc.views > ${l}:`, e, await rpc(`/ctc/views/${l}`, ctcAlice, x, y));
+  };
+
+  const checkViewBytes = async (l1, l2, e) => {
+    await assertEq(`ctc.v     > ${l1} + ${l2}:`, e, [ await rpc(`/ctc/v/${l1}`,     ctcAlice),
+                                                      await rpc(`/ctc/v/${l2}`,     ctcAlice) ]);
+    await assertEq(`ctc.views > ${l1} + ${l2}:`, e, [ await rpc(`/ctc/views/${l1}`, ctcAlice),
+                                                      await rpc(`/ctc/views/${l2}`, ctcAlice) ]);
+  };
+
+  /* Exports */
+  // TODO: how should we access nested elements in arrays?
```

Comment:
I don't understand what this means

Reply:
Maybe you mean that we currently have a way of turning paths like `/a/b/c` into object references like `a.b.c` and you're wondering about how to do `a.b[c]`?

Reply:
Yup, we've now got a way to do `/a/b/c` for `a.b.c` if it's just a nested object of objects, but if `b = [ 'c', { d: 'e' } ]`, how should we pluck out `a.b[0]` or `a.b[1]`? (Or `d`?)

Reply:
I think you need to make up something or look for precedent like XPath, JsonPath, or the Go template syntax, etc. FWIW, I think JS weirdness means that `/a/b/1/d` will "just work"